### PR TITLE
hotfix(internals): reducer accumulation

### PR DESCRIPTION
**Pull request by justingreenberg**
@mnemanja @goldo @Dattaya can you guys confirm this fix for reducer accumulation bug? it's a one-liner so you don't really even need to pull down the code

**app/utils/asyncInjectors.js**
```js
 // Line 36
 if (Reflect.has(store.asyncReducers, name)) return;
```

@gihrig @mxstbr may want to include this hotfix in v3.2 (#948) if it resolves the bug (#949)... this includes internal template updates and shouldn't affect coverage.

### CODE REVIEW

**Comment by goldo** [comment]
Sorry I am currently working on a July version, and I don't have the file update of August 3rd:
https://github.com/justingreenberg/react-boilerplate/commit/96e9441838ba2ff372e8f3ef490e7fa38d29974c

Testing that in a fresh new react-boilerplate would be easy but I have currently no time :(

**Comment by Dattaya** [comment]
:+1: Now I don't experience this bug anymore (Win10), tested in Chrome, FF, Edge stable, IE11. Couldn't test in IE10 and 9, this boilerplate does't work in them for now.

**Comment by mxstbr** [comment]
LGTM, @gihrig?

File: app/utils/asyncInjectors.js [hunk 1]
```diff
@@ -33,6 +33,8 @@ export function injectAsyncReducer(store, isValid) {
       '(app/utils...) injectAsyncReducer: Expected `asyncReducer` to be a reducer function'
     );
 
+    if (Reflect.has(store.asyncReducers, name)) return;
```
Comment (smashercosmo) [inline app/utils/asyncInjectors.js]: What's the benefit of using Reflect.has instead of hasOwnProperty?

Comment (mxstbr) [inline app/utils/asyncInjectors.js]: Good question, I'm not sure either! @justingreenberg?

**Comment by mnemanja** [comment]
I can confirm that the fix works. Good job @justingreenberg

**Comment by mxstbr** [comment]
Merged, will be included in v3.2! Thanks for testing @mnemanja @Dattaya!

**Comment by smashercosmo** [comment]
So, no answer about Reflect.has (((

File: app/utils/asyncInjectors.js (hunk 1 above)
Comment (justingreenberg) [inline app/utils/asyncInjectors.js]: @mxstbr @smashercosmo sorry guys just getting on my computer... I have been using Reflect mainly as a first-class replacement for `in` and `delete` operators...

> In ES5, the way to detect whether an object obj defines or inherits a certain property name is to write (name in obj). Similarly, to delete a property, one uses delete obj[name]. While dedicated syntax is nice and short, it also means you must explicitly wrap these operations in functions when you want to pass the operation around as a first-class value.

> With Reflect, these operations are readily defined as first-class functions: Reflect.has(obj, name) is the functional equivalent of (name in obj) and Reflect.deleteProperty(obj, name) is a function that does the same as delete obj[name].

there are other good reasons to use Reflect api, check out the wiki: [Harmony `Reflect`](https://github.com/tvcutsem/harmony-reflect/wiki#harmony-reflect)

**Pull request merged**